tests/js/spec/views/performance/transactionSpans/spanDetails.spec.tsx

```diff
@@ -362,6 +364,60 @@ describe('Performance > Transaction Spans > Span Summary', function () {
         expect(searchBarNode).toBeInTheDocument();
       });
 
+      it('disables reset button when no min or max query parameters were set', function () {
+        const data = initializeData({
+          features: FEATURES,
+          query: {project: '1', transaction: 'transaction'},
+        });
+
+        render(<SpanDetails params={{spanSlug: 'op:aaaaaaaa'}} {...data} />, {
+          context: data.routerContext,
+          organization: data.organization,
+        });
+
+        const resetButton = screen.getByRole('button', {
+          name: /reset view/i,
+        });
+        expect(resetButton).toBeInTheDocument();
+        expect(resetButton).toBeDisabled();
+      });
+
+      it('enables reset button when min and max are set', function () {
+        const data = initializeData({
+          features: FEATURES,
+          query: {project: '1', transaction: 'transaction', min: 10, max: 100},
+        });
+
+        render(<SpanDetails params={{spanSlug: 'op:aaaaaaaa'}} {...data} />, {
+          context: data.routerContext,
+          organization: data.organization,
+        });
+
+        const resetButton = screen.getByRole('button', {
+          name: /reset view/i,
+        });
+        expect(resetButton).toBeEnabled();
+      });
+
+      it('clears min and max query parameters when reset button is clicked', function () {
+        const data = initializeData({
+          features: FEATURES,
+          query: {project: '1', transaction: 'transaction', min: 10, max: 100},
+        });
+
+        render(<SpanDetails params={{spanSlug: 'op:aaaaaaaa'}} {...data} />, {
+          context: data.routerContext,
+          organization: data.organization,
+        });
+
+        const resetButton = screen.getByRole('button', {
+          name: /reset view/i,
+        });
+        resetButton.click();
+        expect(browserHistory.push).not.toHaveProperty('min');
+        expect(browserHistory.push).not.toHaveProperty('max');
```

Comment:
It looks like these assertions will pass trivially.

I think `expect(browserHistory.push).toHaveBeenCalledWith()` without `min` and `max` present in the query string would be sufficient.

Reply:
kk, done 👍